### PR TITLE
Document infectious decay window.

### PR DESCRIPTION
**Pull request by beshaya**
Closes: #664

### CODE REVIEW

File: src/posts/paper/9-advanced-method.ts [hunk 1]
```diff
@@ -36,8 +38,7 @@ The best way to understand how to apply the Advanced Method is through an exampl
...
 [^12]:
-     The reason we care most about their activities between 2–9 days ago is because that is the most likely range of time intervals between infections. ([He et al](https://www.nature.com/articles/s41591-020-0869-5), figure 1c middle graph; [Ferretti et al](https://science.sciencemag.org/content/368/6491/eabb6936) “generation time”). See [Research Sources](14-research-sources) for graphs and more about the infectious period.
-
+     The 2-9 day window is an approximation for people who keep relatively constant schedules or maintain a similar level of risky behaviors week over week (for instance, by using microCOVID to track their own activites). If this is not a good description of the person's behavior (e.x. they recently took a flight, went to a party or indoor restaurant, or had contact with someone with COVID), it is necessary to count up all their activities from the last 2-23 days (3 weeks). Events past 5 days have diminishing effects on their riskiness, which makes this method difficult to do by hand. The [Risk Tracker](/tracker) handles this calculation for us. For details on how this is derived, see [Research Sources](/paper/14-research-sources#infectious-period).
```
Comment (blanchardjeremy) [inline src/posts/paper/9-advanced-method.ts:41]: Nit: I think it should be "ex:" or "e.g." or "i.e." but not "e.x."

Comment (catherio) [reply src/posts/paper/9-advanced-method.ts]: agree, typofixed

**Comment by catherio** [comment]
made some wording modifications that sound better to me, but you don't have to keep these (or convince me of anything if you're just like "eh/nah" and want to roll them back); fully defer to you on all further launch of this!

**Review by catherio — APPROVED** [review]
LGTM

**Pull request merged**